packages/xsnap/src/xsnap.js

```diff
@@ -121,7 +163,25 @@ export function xsnap(options) {
         xsnapProcess.kill();
         throw new Error('xsnap protocol error: received empty message');
       } else if (message[0] === OK) {
-        return message.subarray(1);
+        let compute = NaN;
```

Comment:
`NaN` is a pain to deal with. In this case, it means "unknown", right? As in, the child didn't tell us? Could it be simpler if we require the child to always tell us? And/or use `undefined` to mean "unknown"?

Reply:
> And/or use `undefined` to mean "unknown"?

I will use `null` since that is JSON-serializable.